docs/Data_Sources/add_data-source.md

```diff
@@ -40,3 +40,24 @@ The credential information varies by each Data Source type. Once you've filled i
 If the connection is successfully tested, you can add the Data Source by pressing the "Add Data Source" button on the bottom.
 
 ![Test Connection](/img/connecting-to-data-sources/test_connection.png)
+
+## Metadata sync
+
+![Screenshot of a pop-up modal with the title "Data Source Added Successfully" and a line of text below it saying "Please wait for a few minutes for data sync to complete before adding a KPI".](/img/connecting-to-data-sources/metadata-sync-popup.png)
+
+Metadata here refers to all the schemas, tables and columns present in a data source. This data is used to help you add a KPI. When a new data source is added, and every day hence, the metadata of the data source is cached inside Chaos Genius. The time at which metadata is synchronized daily can be [set globally](/Operator_Guides/Configuration/config-params.md#general-parameters).
```

Comment:
```suggestion
Once you've added your data source, give Chaos Genius a couple of minutes to sync it's metadata.

Metadata here refers to all the schemas, tables and columns present in a data source. This data is used to help you add a KPI. When a new data source is added, and every day hence, the metadata of the data source is cached inside Chaos Genius. The time at which metadata is synchronized daily can be [set globally](/Operator_Guides/Configuration/config-params.md#general-parameters).
```

Reply:
The meta data line seemed too abrupt. Just added an intro to it, so that it has a better flow.